Vungle/VungleBannerCustomEvent.m

```diff
@@ -33,6 +33,11 @@ - (BOOL)enableAutomaticImpressionAndClickTracking
 
 - (void)requestAdWithSize:(CGSize)size customEventInfo:(NSDictionary *)info adMarkup:(NSString *)adMarkup
 {
+    if ([[[UIDevice currentDevice] systemVersion] compare:@"10.0" options:NSNumericSearch] == NSOrderedAscending) {
```

Comment:
@crbinsf 
We decided go with the fix from MoPub side, didn't we? Please verify on this.

Reply:
@ashinagawa @crbinsf Hi Aki and Clarke, this branch` 6.6.x_not_server_banner_ios9` and `6.6.x_pre_release` branch already merged the fix from MoPub side:

if (@available(iOS 10.0, *)) {
        self.timeOutTimer = [NSTimer scheduledTimerWithTimeInterval:BANNER_TIMEOUT_INTERVAL repeats:NO block:^(NSTimer * _Nonnull timer) {
        if (!self.isAdCached) {
            [[VungleRouter sharedRouter] clearDelegateForRequestingBanner];
        }
    }];

I have a concern about above MoPub fix, if server side still serves mraids ads on iOS 9 devices(which are not supposed to serve, the server side should have a fix for this issue), this fix will still approve Adapter requesting Banner ads on iOS 9 devices, which will cause unpredictable behaviors because no timeout Timer works.
So I have added this protection in the MoPub Adapter, if the device iOS version is less than 10.0, the MoPub Adapter will reject to request Banner ad. Thanks a lot. ^_^

Reply:
@YueVungle please hold on merging this PR until we have a resolution in the following JIRA ticket: https://vungle.atlassian.net/browse/DP-7279. Thanks.